xarray/tests/test_plot.py

```
@@ -687,6 +687,13 @@ def test_discrete_colormap_list_levels_and_vmin_or_vmax(self):
         assert primitive.norm.vmax == max(levels)
         assert primitive.norm.vmin == min(levels)
 
+    def test_colormap_pass_norm(self):
+        norm = mpl.colors.LogNorm(0.1, 1e1)
+        primitive = self.darray.plot(norm=norm, vmin=2, vmax=5)
```

Should we raise an error in this case instead?